### PR TITLE
Initial add Dutch language

### PR DESCRIPTION
**Pull request by Sateetje**

### CODE REVIEW

File: src/languages/index.js [hunk 1]
```diff
@@ -4,4 +4,9 @@ module.exports = [
         key: "en",
         table: require("./en"),
     },
+    {
+        name: "Nederlands",
```
Comment (saleemrashid) [inline src/languages/index.js:8]: Shouldn't this be "Dutch"?

Comment (Sateetje) [reply src/languages/index.js:8]: Not in my opinion. Dutch is Dutch is Nedelands. So if someone couldn’t read English, he could choose the right language.

**Comment by Sateetje** [comment]
Dutch in Dutch is Nedelands.

**Comment by Sateetje** [comment]
Second typo :(. Too small keyboard on iphones

Dutch in Dutch is Nederlands

**Comment by saleemrashid** [comment]
@Sateetje That makes sense.

I need to make a few changes and then I will merge this, thanks!

File: src/languages/nl.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+module.exports = {
+    TITLE_OFFLINE_WARNING: "Gebruik een offline computer",
+    CONTENT_OFFLINE_WARNING: "Mits je dit nog niet hebt gedaan, download deze pagina en gebruik hem op een offline computer om je persoonlijke sleutels veilig te houden!",
+
+    LABEL_APP_LANGUAGE: "Taal",
+
+    TITLE_RECOVER_WALLET: "Vul 'mnemonic' in om de NEM accounts te herstellen",
+    LABEL_NETWORK: "Netwerk",
+    LABEL_BIP39_MNEMONIC: "BIP-39 'mnemonic'",
```
Comment (saleemrashid) [inline src/languages/nl.js:9]: People seem to use the word "ezelsbruggetje" to refer to BIP-39 mnemonics. Is there any reason not to use it here?

File: src/languages/index.js [hunk 3]
```diff
@@ -4,4 +4,9 @@ module.exports = [
...
+        key: "nl",
+        table: require("./nl"),
+    },
```
Comment (saleemrashid) [inline src/languages/index.js:11]: Please could you remove the extra space at the end of this line

**Review by saleemrashid — CHANGES_REQUESTED** [review]
Checked with Google Translate and they look good! Two small nits though :wink:

**Comment by saleemrashid** [comment]
Merged and squashed in https://github.com/saleemrashid/nem-trezor-standalone/commit/e4383a9a422c0922d7f3f18b39001e89ea910f37

Not sure about translation of mnemonic though, so another PR changing it to "ezelsbruggetje", if that's correct, would be great!

**Comment by Sateetje** [comment]
Translation of mnemonic could be 'herstel woorden'. 'Ezelsbruggetje' isn't the right translation for this use.

**Pull request closed without merging**